node/heartbeat/hearbeatMessageInfo_test.go

```diff
@@ -72,17 +73,20 @@ func TestHeartbeatMessageInfo_HeartbeatShouldUpdateUpTime(t *testing.T) {
 
 	hbmi, _ := newHeartbeatMessageInfo(time.Duration(10), false)
 	incrementalTime := int64(0)
+	mut := sync.Mutex{}
 	hbmi.getTimeHandler = func() time.Time {
+		mut.Lock()
 		tReturned := time.Unix(0, incrementalTime)
 		incrementalTime += 1
-
+		mut.Unlock()
```

Comment:
do we keep the mutex since there is also a sleep?

Reply:
yes, no need for the mutex, since there is a 10 ms sleep between calls.